AZ3166/src/cores/arduino/cli/console_cli.cpp

```diff
@@ -4,11 +4,15 @@
 #include "mbed.h"
 #include "mico.h"
 #include "mbedtls/version.h"
+#include "mbedtls/base64.h"
+#include "mbedtls/md.h"
+#include "mbedtls/md_internal.h"
 #include "EEPROMInterface.h"
 #include "SystemWiFi.h"
 #include "SystemVersion.h"
 #include "UARTClass.h"
 #include "console_cli.h"
+#include "azure_c_shared_utility/connection_string_parser.h"

```

Comment:
Actually, the DevKit CLI command is not supposed to depend on the Azure IoT Device SDK.
You'd better remove this dependency and use the string API from C